examples/complete/fixtures.us-east-1.tfvars

```diff
@@ -0,0 +1,13 @@
+region = "us-east-1"
+
+namespace = "eg"
+
+name = "application"
+
+stage = "development"
+
+domain = "example.com"
+
+verify_domain = true
```

Comment:
since it's set to `true` and the domain name is `example.com`, will it be able to verify that domain (which we don't control)?

Reply:
No, it won't verify it but it will create DNS records.
If we have valid DNS domain on that account then I could use that instead of private DNS zone. 
Also only valid test for this module to send email which is rather hard to do in `terratest`.